root.tf

```diff
@@ -490,13 +490,15 @@ module "export_bucket" {
   project     = var.project
   function    = "consignment-export"
   common_tags = local.common_tags
+  kms_key_id  = module.s3_external_kms_key.kms_key_arn
```

Comment:
Only apply the encryption key to the judgment bucket, not the standard bucket. For the standard bucket need to ensure that the Digital Archivists can still access it after encryption, otherwise they can't process the standard transfers

Reply:
I have removed the encryption key, thanks